sentry_sdk/scope.py

```diff
@@ -122,7 +123,44 @@ def __init__(self):
         self._propagation_context = None  # type: Optional[Dict[str, Any]]
 
         self.clear()
-        self.generate_propagation_context()
+
+        incoming_trace_information = self._load_trace_data_from_env()
+        self.generate_propagation_context(incoming_data=incoming_trace_information)
+
+    def _load_trace_data_from_env(self):
+        # type: () -> Optional[Dict[str, str]]
+        """
+        Load Sentry trace id and baggage from environment variables.
+        Can be disabled by setting SENTRY_USE_ENVIRONMENT to "false".
+        """
+        incoming_trace_information = None
+
+        false_values = [
+            "false",
+            "no",
+            "off",
+            "n",
+            "0",
+        ]
```

Comment:
Maybe we could put this list someplace else (`utils.py`?) and also use it here https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/_compat.py#L85, so that we don't have multiple definitions of what falsy means in different places

Reply:
good idea!